### DISCUSSION TITLE
jupyter-book - customizing book-theme for single page reports

### DISCUSSION THREAD

**Discussion by sbaker-at-work** [executablebooks/meta#790] [Q&A]
I've been using Jupyter Book for documentation in circuit design, and I'm really loving it - it's waaaay better than massaging data in Excel then copying & pasting a bunch of tables and plots into Word!

One issue I'm having is with single page reports - I can't find a way to get rid of the left navbar, which is empty for a single page report.

I've tried:
- using the flow for building an article, documented here: [link](https://jupyterbook.org/en/stable/structure/toc.html#build-an-article-from-a-single-file)
- using the article flow, with `--builder singlehtml`
- using the article flow, with the book-theme config option documented here: [link](https://sphinx-book-theme.readthedocs.io/en/stable/customize/single-page.html)
    - somewhat related thread: #285 

Each time, I still get the big empty area on the left of my page.

Am I missing something, or is this working as intended?

Thanks!

![image](https://user-images.githubusercontent.com/110132685/181376082-34427b41-2e6e-45cb-bdd9-6f5d527a111a.png)

**Comment by choldgraf** [comment]
Would you expect it to snap to the left of the page instead of being centered? I think the centering is intended so you may need to make it snap to the left with some custom CSS or something

**Reply by sbaker-at-work** [reply]
Thanks for the quick reply @choldgraf !

After playing around with Sphinx standalone, I realized that the article building flow is actually using the 'single_page : True' config setting, and things are working as expected.

> Would you expect it to snap to the left of the page instead of being centered?

Yeah, that's what I was expecting, but I can see how folks might like it the way it is.

I'll try out some CSS to shrink the margin a bit, but since this is really outside my wheelhouse, I would love any suggestions that you may have!

**Reply by sbaker-at-work** [reply]
Wow, this was way easier than I would've guessed! All I need in a CSS file is:

`#site-navigation { display: none;}`

I didn't see it in the docs, but is there any way I could point Jupyter Book to a custom CSS file via the command-line? Or do I always need to run the build from a directory where I've got this CSS inside the _static folder?

Again, thanks for the help, and for this awesome tool!
